### PR TITLE
Docs WG Agenda/Minutes -- February 2022

### PR DESCRIPTION
**Pull request by encukou**
Fixes: #15

Note: I know this'll be hard for some of us, but I think it's not worth it to spend too much energy on formatting/proofreading the meeting notes. There are much better ways to spend time & energy :)

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]

File: docs/monthly-meeting/index.rst [hunk 1]
```diff
@@ -11,4 +11,4 @@ Monthly reports in reverse chronological order.
    :maxdepth: 1
    :caption: Monthly reports
 
-   May 2021 (TBD) <2021-05.md>
\ No newline at end of file
+   Feb 2022 <2022-02.md>
```
Comment (pradyunsg) [inline docs/monthly-meeting/index.rst:14]: Can we instead add a `:glob:` option and add a single * here?

That, combined with a document heading change, would mean that future meeting notes PRs will only need to just add a file.

**Review by pradyunsg — APPROVED** [review]
Haven't looked carefully, but it certainly makes sense to put these up publicly!

One minor non-blocking suggestion, to simplify future PRs for these notes. :)

Thanks for doing this @encukou!

File: docs/monthly-meeting/index.rst (hunk 1 above)
Comment (encukou) [reply docs/monthly-meeting/index.rst:14]: Sounds good! I'll leave it to a future PR, maybe someone with more Sphinx knowledge will take it :)

**Comment by encukou** [comment]
Thanks for the reviews!

**Pull request merged**